app/jobs/autotune/build_job.rb

```diff
@@ -71,7 +62,7 @@ def perform(project, target: 'preview', current_user: nil)
       end
 
       # Set status and save project
-      project.published_at = DateTime.current if target.to_sym == :publish
+      project.update_published_at = true if target.to_sym == :publish
```

Comment:
rename this to `update_published` instead of `update_published_at` since it is boolean?

Reply:
I think I called it `update_published_at` because it triggers an update of the `published_at` field. It's kind of a weird thing I had to do to make sure the `published_at` and `data_updated_at` fields were synchonized.

Reply:
they should probably all be named `_time` instead of `_at`